Sevenpros/Teamwork
# #168373806 build out specific article view

#### What does this PR do?
create popup modal to view the specific article
#### Description of Task to be completed?
display modal on button click
adding comment area below the article text
add post button to submit the comment
#### How should this be manually tested?
when a user click on specific article she/he should see 
the popup modal containing the whole article test he/she can see the comment area too


#### What are the relevant pivotal tracker stories?
 #168373806 #168373696




[Finishes #168373806]
